src/Websites/Websites/Cmdlets/DeploymentSlots/NewAzureWebAppSlot.cs

```diff
@@ -100,6 +100,9 @@ public class NewAzureWebAppSlotCmdlet : WebAppBaseClientCmdLet
 		[Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
         public SwitchParameter AsJob { get; set; }
```

Comment:
```suggestion
		public SwitchParameter AsJob { get; set; }
```
Correct indent please